packages/storybook/src/spright/chat/conversation/chat-conversation.stories.ts

```diff
@@ -115,6 +116,11 @@ export const chatConversation: StoryObj<ChatConversationArgs> = {
         </${chatConversationTag}>
     `),
     argTypes: {
+        hideBackground: {
+            description:
```

Comment:
We use the attribute names rather than the property names in storybook
```suggestion
            name: 'hide-background',
            description:
```

Reply:
Attribute name has changed.